src/main.rs

```diff
@@ -168,6 +171,25 @@ fn handle_shp(file_path: &str, simplification: f64, is_area: bool) -> Vec<GridGe
         .collect()
 }
 
+fn handle_wkt(input_str: String) -> Vec<GridGeom<f64>> {
```

Comment:
This should match the signature of `handle_topojson` where we're accepting both `simplification` and `is_area` and then passing them to `GridGeom::vec_from_geom` for processing the geometries